docs/source/getting_started/active_learning.rst

```diff
@@ -157,8 +157,9 @@ Use this list to get predictions on the unlabeled images.
 **Important:** The predictions need to be in the same order as the filenames in the
 list returned by the `ActiveLearningAgent`.
 
-For classification, the predictions need to be in a numpy array and normalized,
-such that the rows sum to one. Then, create a scorer object like so:
+Next, create a scorer object depending on your task (see :ref:`lightly-active-learning-scorers` below).
+For instance, for classification, the predictions need to be in a numpy array and normalized,
+such that the rows sum to one:
```

Comment:
The text here was written just for classification before introducing scorers. Rewrote to be more general.